test/chi/components/picker-group.pug

```diff
@@ -15,39 +15,80 @@ title: Picker group
   h3 Base checked
   .m-pickerGroup
     each type in ['Radio1','Radio2','Radio3']
-      input.a-picker(type="radio", name='box', id=`i-${type}`, checked=(type==='Radio2'))
-      label(for=`i-${type}`)
+      input.a-picker(type="radio", name='box', id=`ia-${type}`, checked=(type==='Radio2'))
+      label(for=`ia-${type}`)
         = type
+.test-with-texts-and-icons
+  h3 Texts and icons
+  .m-pickerGroup
+    input.a-picker(type="radio", name='radios', id=`ib-radio1`)
+    label(for=`ib-radio1`)
+      i.a-icon.icon-atom
+      span
+        = `Radio1`
+    input.a-picker(type="radio", name='radios', id=`ib-radio2`)
+    label(for=`ib-radio2`)
+      span
+        = `Radio2`
+      i.a-icon.icon-atom
+    input.a-picker(type="radio", name='radios', id=`ib-radio3`)
+    label(for=`ib-radio3`)
+      i.a-icon.icon-atom
+      span
+        = `Radio3`
```

Comment:
@janibolkvadze can you remove this line and move to line 37 with this format span Radio 3